src/query/query_parser/query_grammar.rs

```diff
@@ -44,9 +44,9 @@ parser! {
         };
         let term_val_with_field = negative_number().or(term_val());
         let term_query =
-            (field(), term_val_with_field)
-            .map(|(field_name, phrase)| UserInputLiteral {
-                field_name: Some(field_name),
+            (optional(field()), term_val_with_field)
```

Comment:
What is the point of this change? 
I don't think this works. Besides, the absence of fields is handled below.

Reply:
reverted my change